### PR TITLE
added include_attachments

### PR DESCRIPTION
**Pull request by MoisesRT**
I added an option to include attachments while leaving intact for backwards compatibility.

### CODE REVIEW

**Comment by blag** [comment]
I don't know that we need to make this a config option, just start including them.

Adding payload keys shouldn't break anything. However, if we do add the `attachments` payload key then we should bump at least the minor version number of this pack.

@LindsayHill What do you think?

**Comment by MoisesRT** [comment]
this PR might have merge conflicts with my other PR so if its gets approved and merged ill fix it here and then update the version.

regarding adding always I agree, I did it as backwards compatible but I also agree.

**Comment by MoisesRT** [comment]
I modified the include attachments to be included always also merged with allow_bot_messages and updated the bumped the version
@blag 
@LindsayHill

File: sensors/slack_sensor.py [hunk 1]
```diff
@@ -158,6 +162,13 @@ def _handle_message(self, data):
             # Note: We resolve user and channel information to provide more context
             user_info = self._get_user_info(user_id=data['user'])
 
+        # Check if the message was written by a bot without as_user attribute
+        if self._allow_bot_messages and 'user' not in data and 'username' in data:
+            user_info = {'name': data['username'], 'profile': {}}
+        else:
+            # Note: We resolve user and channel information to provide more context
+            user_info = self._get_user_info(user_id=data['user'])
+
```
Comment (blag) [inline sensors/slack_sensor.py]: This line contains whitespace - that's why the build failed: https://circleci.com/gh/StackStorm-Exchange/stackstorm-slack/110#action-104

Please remove it.

Comment (MoisesRT) [reply sensors/slack_sensor.py]: wired, its like that master too. its not a line I changed
https://github.com/StackStorm-Exchange/stackstorm-slack/blob/master/sensors/slack_sensor.py#L171

Comment (MoisesRT) [reply sensors/slack_sensor.py]: Done

**Comment by blag** [comment]
Thanks!

**Pull request merged**